plenum/server/req_authenticator.py

```diff
@@ -0,0 +1,51 @@
+from typing import Optional
+
+from plenum.common.constants import TXN_TYPE
+from plenum.common.exceptions import NoAuthenticatorFound
+from plenum.common.types import OPERATION
+from plenum.server.client_authn import ClientAuthNr
+
+
+class ReqAuthenticator:
```

Comment:
How are we going to use this class?

Reply:
This is used to register different authenticators by plugins or otherwise, it is being used in `defaultAuthNr`